package.json

```diff
@@ -15,22 +15,21 @@
     "serve": "docusaurus serve",
     "clear": "docusaurus clear",
     "lint": "standard",
-    "lint:fix": "standard --fix",
+    "lint:fix": "standard --fix && prettier --write scripts src versioned_docs versioned_sidebars *.js",
```

Comment:
I don't think we need to run Prettier on old docs. just to avoid potentially humongous PRs in the future to old docs when we upgrade it.